feat: Add wallets

AkhileshThite opened the pull request:
![Screenshot 2021-09-14 at 10 37 32 PM](https://user-images.githubusercontent.com/68826419/133309655-34a2583b-74e2-43ad-9393-dce89214b748.png)

armaniferrante commented:
Can we use https://github.com/solana-labs/wallet-adapter instead? It should eliminate a lot of boilerplate.

Aadhinana commented:
#85  I tried doing this in with `wallet-adpater`. Let me know what you think. :smile:

AkhileshThite commented:
> Can we use https://github.com/solana-labs/wallet-adapter instead? It should eliminate a lot of boilerplate.

Added multiple wallets by using solana wallet adapter, removed old connect button. Let me know if you want any further changes.
https://user-images.githubusercontent.com/68826419/133996856-e9cb3d9b-0107-49e5-aa1f-e828a2415d0a.mp4
